test/pluginator/name_converter_test.rb

```diff
@@ -29,5 +29,13 @@ class Converter
     it :builds_class do
       Converter.send(:name2class, "Converter").must_equal(Converter)
     end
+
+    it :builds_class_with_dash do
+      Converter.send(:name2class, "puppet-debugger").must_equal(PuppetDebugger)
```

Comment:
the tests are failing because of missing `PuppetDebugger`, maybe add the class/module on top?